src/taxonomy/modals/ExportModal.test.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { IntlProvider, injectIntl } from '@edx/frontend-platform/i18n';
+import { initializeMockApp } from '@edx/frontend-platform';
+import { AppProvider } from '@edx/frontend-platform/react';
+import { render, fireEvent } from '@testing-library/react';
+import ExportModal from './ExportModal';
+import initializeStore from '../../store';
+import { callExportTaxonomy } from '../api/hooks/selectors';
+
+const onClose = jest.fn();
+let store;
+const taxonomyId = 1;
+
+jest.mock('../api/hooks/selectors', () => ({
+  callExportTaxonomy: jest.fn(),
+}));
+
+const ExportModalComponent = () => (
+  <AppProvider store={store}>
+    <IntlProvider locale="en" messages={{}}>
+      <ExportModal intl={injectIntl} taxonomyId={taxonomyId} onClose={onClose} isOpen />
+    </IntlProvider>
+  </AppProvider>
+);
+
+describe('<ExportModal />', async () => {
+  beforeEach(async () => {
+    initializeMockApp({
+      authenticatedUser: {
+        userId: 3,
+        username: 'abc123',
+        administrator: true,
+        roles: [],
+      },
+    });
+    store = initializeStore();
+  });
+
+  it('should render the modal', () => {
+    const { getByText } = render(<ExportModalComponent />);
+    expect(getByText('Select format to export')).toBeInTheDocument();
+  });
+
+  it('should call export endpoint', () => {
+    const { getByText } = render(<ExportModalComponent />);
+
+    fireEvent.click(getByText('JSON file'));
+    fireEvent.click(getByText('Export'));
+
+    expect(onClose).toHaveBeenCalled();
+    expect(callExportTaxonomy).toHaveBeenCalledWith(taxonomyId, 'json');
+  });
+});
```

Comment:
IMO these tests are not necessary since they are testing react itself not our code.

Reply:
Can we keep this tests? I added this test to cover the selection of a non default format (JSON in this case) and avoid coverage issues

Reply:
This is the problem I have with automated coverage checks... they often make you write tests that aren't actually useful. But yeah, if we're testing some non-default functionality or we're required to put pointless tests in to appease the coverage bot, then leave them in.

Reply:
I think in that case you can go one level up and test whatever is consuming this component and not mock out this component. Go up far enough till you reach a component that has testable logic, and test that. Since this code will be used by it it will be covered in the tests.

Reply:
@xitij2000 Thanks for the recommendation. I updated this here 1abf0342616a8c0c4167a5209afb3c7abdab02ac